install/hoarder-install.sh

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: MickLesk (Canbiz) & vhsdream
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y \
+  python3 \
+  python3-distutils \
+  g++ \
+  build-essential \
+  curl \
+  git \
+  sudo \
+  gnupg \
+  ca-certificates \
+  chromium \
+  libpixman-1-dev \
+  libcairo2-dev \
+  libjpeg-dev \
+  libpango1.0-dev \
+  libgif-dev \
+  librsvg2-dev \
+  pkg-config \
+  mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Additional Tools"
+wget -q https://github.com/Y2Z/monolith/releases/latest/download/monolith-gnu-linux-x86_64 -O /usr/bin/monolith
+chmod +x /usr/bin/monolith
+wget -q https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp_linux -O /usr/bin/yt-dlp
+chmod +x /usr/bin/yt-dlp
+msg_ok "Installed Additional Tools"
+
+msg_info "Installing Meilisearch"
+cd /tmp
+wget -q https://github.com/meilisearch/meilisearch/releases/latest/download/meilisearch.deb
+$STD dpkg -i meilisearch.deb
+wget -q https://raw.githubusercontent.com/meilisearch/meilisearch/latest/config.toml -O /etc/meilisearch.toml
+MASTER_KEY=$(openssl rand -base64 12)
+sed -i \
+    -e 's|^env =.*|env = "production"|' \
+    -e "s|^# master_key =.*|master_key = \"$MASTER_KEY\"|" \
+    -e 's|^db_path =.*|db_path = "/var/lib/meilisearch/data"|' \
+    -e 's|^dump_dir =.*|dump_dir = "/var/lib/meilisearch/dumps"|' \
+    -e 's|^snapshot_dir =.*|snapshot_dir = "/var/lib/meilisearch/snapshots"|' \
+    -e 's|^# no_analytics = true|no_analytics = true|' \
+    /etc/meilisearch.toml
+msg_ok "Installed Meilisearch"
+
+msg_info "Installing Node.js"
+mkdir -p /etc/apt/keyrings
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_22.x nodistro main" >/etc/apt/sources.list.d/nodesource.list
+$STD apt-get update
+$STD apt-get install -y nodejs
+msg_ok "Installed Node.js"
+
+msg_info "Installing Hoarder"
+cd /opt
+RELEASE=$(curl -s https://api.github.com/repos/hoarder-app/hoarder/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+wget -q "https://github.com/hoarder-app/hoarder/archive/refs/tags/v${RELEASE}.zip"
+unzip -q v${RELEASE}.zip
+mv hoarder-${RELEASE} /opt/hoarder
+
+mkdir -p /opt/hoarder_data
+
+cd /opt/hoarder
+corepack enable
+export PUPPETEER_SKIP_DOWNLOAD="true"
```

Comment:
are these not needed in update function?

Reply:
Nodejs only needs to be installed the one time. As for yt-dlp, Monolith and Meilisearch, it might be a good idea to also install newer versions during the update script, but I'm not sure if that is standard protocol.